aosp_diff/preliminary/external/drm_hwcomposer/0002-Fix-issue-have-OVERLAY-crtc-count-neq-connector-count.patch

```diff
@@ -0,0 +1,212 @@
+From ba5bcbbcf8a09eaf84c95e86f635846f10eb2abf Mon Sep 17 00:00:00 2001
+From: "Li, HaihongX" <haihongx.li@intel.com>
+Date: Mon, 2 Aug 2021 10:00:08 +0800
+Subject: [PATCH] Fix issue while crtc count not equal to connector count,
+ meanwhile there is overlay plane
```

Comment:
comments are not enough, you have done multiple changes:
1, SetActiveConfigWithConstraints change
2, crtc count and connector count doesn't match
3, only provide preferred mode
4, add property to control planes enabling
......
need describe all these changes